soap/src/java/com/zimbra/soap/JaxbUtil.java

```diff
@@ -988,8 +988,8 @@
             com.zimbra.soap.mail.message.SaveDraftResponse.class,
             com.zimbra.soap.mail.message.SaveIMAPSubscriptionsRequest.class,
             com.zimbra.soap.mail.message.SaveIMAPSubscriptionsResponse.class,
-            com.zimbra.soap.mail.message.SaveProfileImageRequest.class,
-            com.zimbra.soap.mail.message.SaveProfileImageResponse.class,
+            com.zimbra.soap.mail.message.ModifyProfileImageRequest.class,
+            com.zimbra.soap.mail.message.ModifyProfileImageResponse.class,
```

Comment:
Perhaps try and retain the sort order of lines

Reply:
Fixed